library/src/armory/model/object_detection/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""This package contains model wrappers for object detection."""
+
+from armory.model.object_detection.jatic_object_detector import (
+    JaticObjectDetectionModel,
+)
```

Comment:
One thing to be wary of when creating package `__init__.py` files like in this case is now that all the imports in the `jatic_object_detector` module are required to be installed regardless of whether you're using it or not. I think in this case we're still OK because the actual jatic-toolbox import only occurs during type checking.